File: Masodik_Labor_Hazi/FizzBuzz.java
public class FizzBuzz {

    private int szam;

    public FizzBuzz(int szam) {
        this.szam = szam;
    }

    public void start() {
        for (int i = 1; i <= szam; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("fizzbuzz");
            } else if (i % 3 == 0) {
                System.out.println("fizz");
            } else if (i % 5 == 0) {
                System.out.println("buzz");
            } else {
                System.out.println(i);
            }
        }
    }

    public static void main(String[] args) {
        FizzBuzz fb = new FizzBuzz(100);
        fb.start();
    }
}

File: C#_hazi/FizzBuzz/FizzBuzz/Fizzbuzz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FizzBuzz
{
    internal class Fizzbuzz
    {

        private int limit;

        public Fizzbuzz(int limit)
        {
            this.limit = limit;
        }

        public void Start()
        {
            for (int i = 0; i < limit; i++)
            {
                if (i % 3 == 0 && i % 5 == 0)
                {
                    Console.WriteLine("fizzbuzz");
                }
                else if (i % 3 == 0)
                {
                    Console.WriteLine("fizz");
                }
                else if (i % 5 == 0)
                {
                    Console.WriteLine("buzz");
                }
                else
                {
                    Console.WriteLine(i);
                }

            }
        }

    }
}
